src/PAModel/PAConvert/PAWriterVisitor.cs

```diff
@@ -47,13 +47,14 @@ public override void Visit(BlockNode node, Context context)
             node.Name.Accept(this, context);
             context._yaml.WriteStartObject(context._sb.ToString());
 
-            
-            foreach (var func in node.Functions)
+            // Ensure the order we write properties is canonical so that we don't get
+            // noisy diffs on multiple exports. 
+            foreach (var func in node.Functions.OrderBy(x => x.Identifier))
```

Comment:
I wanted to enforce ordering in the writer, not the IR, because the IR could be highly mutable. Just do ordering once when we write.